lib/tasks/update_type_column_for_draft_vacancies.rake

```diff
@@ -1,12 +0,0 @@
-namespace :db do
-  desc "update draft vacancies type column"
-  task update_draft_vacancy_type: :environment do
-    Vacancy.draft.find_each do |v|
-      if v.discarded?
-        v.destroy!
-      else
-        v.update_column(:type, "DraftVacancy")
-      end
-    end
-  end
-end
```

Comment:
@scruti I haven't actually run this one yet - doing it Fri 13th June 2025.

Reply:
Oh God, good you spotted it! 
Will hold until you have run it.